server/package.json

```diff
@@ -20,7 +20,8 @@
     "express-ws": "5",
     "immutable": "4",
     "lodash": "4",
-    "msgpack-lite": "0.1"
+    "msgpack-lite": "0.1",
+    "wrtc": "0.4"
```

Comment:
This is creating some issues on M1 Mac...
https://github.com/node-webrtc/node-webrtc/issues/698

Reply:
I can confirm; they don't offer macOS M1 binaries. A possible work-around (according to the mentioned issue) would be to use Rosetta to install everything for an x64 arch. However, this conflicts with TF.js which does not work with Rosetta.